Add the option to compute the full joint wrench.

SylvainBertrand opened the pull request:
This PR enables the `ForwardDynamicsCalculator` to compute the full joint wrench. The goal is to allow simulation to compute the wrenches across the robot body to do FEA and whatnot.

SylvainBertrand committed:
Add the option to compute the full joint wrench.

james-p-foster reviewed (approved):
LGTM. There's a lot of formatting changes, just curious -- are you doing anything differently with the formatting?

SylvainBertrand commented:
I switched to IntelliJ, sadly I can't get the formatter to perfectly match Eclipse's one. I thought I had reverted these changes though, I'm gonna fix that.

SylvainBertrand committed:
Reducing diff due to formatting.
